static/js/compose.js

```diff
@@ -565,7 +527,6 @@ function send_message(request) {
     if (request === undefined) {
         request = create_message_object();
     }
-    exports.snapshot_message(request);
```

Comment:
Yeah, you're right, we can kill this.  I'll do it in its own commit so there's a clear explanation for why, but basically it became obsolete when we implemented local echo.